controllers/storagecluster/cephobjectstores.go

```diff
@@ -283,3 +287,19 @@ func hasNonPortableOSD(sc *ocsv1.StorageCluster) bool {
 	}
 	return true
 }
+
+// when using non default hostNetwork Object Store should run on hostNetwork and use different port to avoid port collision
+func getPort(sc *ocsv1.StorageCluster) int32 {
+	if shouldUseHostNetworking(sc) {
+		return 31080
+	}
+	return 80
+}
+
+// when using non default hostNetwork Object Store should run on hostNetwork and use different secure port to avoid port collision
+func getSecurePort(sc *ocsv1.StorageCluster) int32 {
+	if shouldUseHostNetworking(sc) {
+		return 31044
```

Comment:
@nb-ohad @sp98  @BlaineEXE @parth-gr @leelavg 
Currently ocs-operator hard codes the port and securePort for CephObjectStores, 
Do we want to enable ocs-operator to provide option to set port and secure port for ObjectStore or we hard code a different port for Data replication use case?

Reply:
Updated, to have option in StorageCluster CR and a fallback if not set